.nais/nais-q1.yml

```diff
@@ -40,8 +40,9 @@ spec:
       - mountPath: /secrets/innsending-data
         kvPath: secret/team-soknad/innsending/basicauth
       - mountPath: /var/run/secrets/nais.io/resend
-        kvPath: /kv/preprod/fss/soknadsmottaker/resend/q1
```

Comment:
Se kommentar over.For ikke å re-sende i feil miljø, legg eventuell resending i miljøspesifikk mappe.